src/ray/stats/metric_defs.cc

```diff
@@ -196,6 +185,43 @@ DEFINE_stats(gcs_storage_operation_count,
              (),
              ray::stats::COUNT);
 
+/// Object store
+/// Local object store memory of different types reported from multiple
+/// components. Reported metric from different subtypes, e.g. Location
+/// and Type could overlap. Reported metric within the same type might
```

Comment:
What do you mean by overlap? The bytes should always sum up to the total number of actual bytes in memory + disk, right?

Reply:
Oh I mean overlapping of bytes from different subgroups.  
1. The definition of the metric group: 
For example, the amount of spillable/evictable/inUse bytes overlap with bytes InMemory ; bytes in PrimaryCopy overlap with bytes in "InUse" and "Spillable". 

2. The reporting sources:
So metrics are exported at 3 places now:  the plasma store backend, the LocalObjectManager (spilling mostly) and the ObjectManager. It will probably require some sort of refactoring to centralize it. 


But yeah, I agree some of the groups should add up to 1, for example
- Total usage = `InMemory` + `Fallback` + `Spilled` 
- `InMemory` = `PrimaryCopy` + `PendingSpill` 
- all bytes from different `Creator` (e.g. `Worker`/`RemoteRaylet`) across nodes should sum up to the total size of `InMemory` + `Spilled` ?

I am 100% sure about these though, cc @stephanie-wang ^